package.json

```diff
@@ -46,6 +46,7 @@
     "react-native": "^0.67.4",
     "react-native-webview": "^11.17.2",
     "typescript": "^4.5.4",
-    "vite": "^2.8.0"
+    "vite": "^5.0.12",
```

Comment:
Running `npm install` doesn't update the lock?

Reply:
What do you mean? The lock file updated to the installed version: `ˆ5.0.12` 👀. Is something missing?

Reply:
I mean the fixup commit didn't change the lock, but if you run `npm install` with the updated version pinning directive it might change. I'm asking if you run `npm install` after adding the caret and nothing changed, or if you just changed the version and didn't check for changes on the lock

Reply:
I'm blind, ignore me